testutil/src/test/java/org/spine3/test/GivenShould.java

```diff
@@ -87,7 +93,36 @@ protected AProjection(String id) {
 
     @Test
     public void create_builder_for_process_managers() {
-        assertEquals(AProcessManager.class, Given.processManagerOfClass(AProcessManager.class).getResultClass());
+        assertEquals(AProcessManager.class, Given.processManagerOfClass(AProcessManager.class)
+                                                 .getResultClass());
+    }
+
+    @Test
+    public void pass_the_null_tolerance_check_in_the_given_class() {
```

Comment:
The method name (in particular, its last part) does not make sense.
It now reads as:
`Given should pass the null-tolerance check in the Given class`.

Don't you think?